src/plugins/gs-flatpak.c

```diff
@@ -2418,6 +2431,25 @@ gs_flatpak_update_app (GsFlatpak *self,
 					    gs_app_get_flatpak_branch (app),
 					    gs_flatpak_progress_cb, app,
 					    cancellable, &local_error);
+	/* workaround for when file objects are missing and a static delta
+	 * cannot be used */
+	if (xref == NULL && error_is_ostree_missing_file_obj (local_error)) {
+		g_warning ("Error due to missing file when updating '%s': %s. "
```

Comment:
@cosimoc My reasoning here was that this should not be reached normally. If it is, we have this workaround in place but it'd still be nice for it to show up in the journal, so we get a notion of whether this is frequently applied when users update. And knowing that is useful after we have a way to prevent the issue IMO.
If you still want to downgrade it to a debug message I understand, just let me know.